elf_reader_test.go

```diff
@@ -674,6 +673,18 @@ func TestLibBPFCompat(t *testing.T) {
 			t.Fatalf("Can't read %s: %s", file, err)
 		}
 
+		switch file {
+		case "test_sk_assign.o":
+			// Test contains a legacy iproute2 bpf_elf_map definition.
+			for _, m := range spec.Maps {
+				if m.Extra.Len() == 0 {
+					t.Fatalf("Expected extra bytes in map %s", m.Name)
+				}
+				_, _ = io.Copy(io.Discard, &m.Extra)
+				m.Pinning = PinByName
```

Comment:
Is pinning required for the test to function?

Reply:
Nope, removed.